### DISCUSSION TITLE
GAP 4.13.1 released

### DISCUSSION THREAD

**Discussion by fingolfin** [gap-system/gap#5742] [General]
Dear all,

I am happy to announce the release of [GAP 4.13.1](https://github.com/gap-system/gap/releases/tag/v4.13.1).

Some people who I know package GAP and might be interested in this include @antonio-rojas @isuruf @saraedum @jamesjer @orlitzky @collares @AndreasEnge @yurivict

**Comment by jamesjer** [comment]
Thank you for the notification.  On my first attempt to build for Fedora Rawhide, one builder reported one failed test:
```
testing: /builddir/build/BUILD/gap-4.13.1-build/gap-4.13.1/tst/testinstall/tes\
tunix/streams.tst
########> Diff in /builddir/build/BUILD/gap-4.13.1-build/gap-4.13.1/tst/t\
estinstall/testunix/streams.tst:13
# Input is:
repeat str := ReadLine(s); until str <> fail; str;
# Expected output:
"The cat sat on the mat\n"
# But found:
"The cat sat on the mat"
########
########> Diff in /builddir/build/BUILD/gap-4.13.1-build/gap-4.13.1/tst/t\
estinstall/testunix/streams.tst:19
# Input is:
repeat str := ReadAllLine(s); until str <> fail; str;
# Expected output:
"abc\n"
# But found:
"\n"
########
     280 ms (279 ms GC) and 74.3KB allocated for testunix/streams.tst
```

I think the second test read the newline that should have been read in the first test.  When I tried to reproduce locally, I could not.  I eventually submitted the build again and this time all tests passed on all builders.  I suspect this means that there is a race somewhere in that test, but I'm not sure where to look.

GAP 4.13.1 is available in Fedora Rawhide now, and will be available as an update for Fedora 40 in about a week.

**Reply by ChrisJefferson** [reply]
The stream tests have had races before -- I will take a look and see if we can make it more resilient for future.
